flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/integration/test/utils/TestUtils.java

```diff
@@ -48,14 +49,20 @@ public static void assertTableResult(
     }
   }
 
-  public static void assertColumns(Column[] expected, Column[] actual) {
+  public static void assertColumns(Column[] expected, Column[] actual, String provider) {
```

Comment:
it's a little odd to pass `provider` to `assertColumns`, how about moving `assertColumns` to `FlinkCommonIT`,  and add an method `bool  defaultValueWithNullLiterals` or a better name with default to false, JDBC override to true.